lib/engine/game/base.rb

```diff
@@ -1128,11 +1128,19 @@ def value_for_dumpable(player, corporation)
         return value_for_sellable(player, corporation) if self.class::PRESIDENT_SALES_TO_MARKET
 
         max_bundle = bundles_for_corporation(player, corporation)
-          .select { |bundle| bundle.can_dump?(player) && @share_pool&.fit_in_bank?(bundle) }
+          .select { |bundle| can_dump?(player, bundle) && @share_pool&.fit_in_bank?(bundle) }
           .max_by(&:price)
         max_bundle&.price || 0
       end
 
+      def can_dump?(entity, bundle)
+        if active_step.respond_to?(:can_dump?)
+          active_step.can_dump?(entity, bundle)
+        else
+          bundle.can_dump?(entity)
+        end
+      end
+
```

Comment:
Yeah, this breaks 1841 because I already have a game method named can_dump? Either the 1841 or this has to change the name of the method.

Reply:
Probably the 1841 game method should be renamed since it uses different parameters than the step versions.